.github/workflows/workflow.yml

```diff
@@ -299,12 +299,19 @@ jobs:
           name: basilisk_runtime.compact.compressed.wasm
           path: scripts/upgrade-runtime
       - uses: dawidd6/action-download-artifact@v2
-        name: Download basilisk binary from master
+        name: Download basilisk binary from master (legacy)
```

Comment:
what is the reasoning here?

Reply:
i think the goal here is to properly test a runtime upgrade from old to new version instead of new->bumped new?